packages/react-button/src/components/Button/Button.stories.tsx

```diff
@@ -1,168 +1,18 @@
 import * as React from 'react';
-import { Button, ButtonProps } from '../../Button';
+import { Button } from '../../Button';
```

Comment:
Silly question: why in `Button*.stories` `Button` is imported as:

```ts
import { Button } from '@fluentui/react-button';
```

But here we have a relative import?

Reply:
Oh well, I have an answer:

![image](https://user-images.githubusercontent.com/14183168/131641511-7bc898c9-6ee0-4cef-9cbc-9ebef2b5efc6.png)

@ling1726 do you remember a solution for this?

Reply:
This is because `yarn build` runs without aliases, the only feasible way I can think of is to exclude `.stories` files from compilation, but that might also result in a lot of other changes

Reply:
https://github.com/microsoft/fluentui/pull/18695#issuecomment-868432982

^ I guess it's a solution 😥

Reply:
kudoes to @layershifter  for finding this gem: https://github.com/microsoft/fluentui/pull/18695#issuecomment-868432982